build.gradle

```diff
@@ -304,8 +309,59 @@ shadowJar {
         //  it will be at the same level as the newly created jar.  (overwriting symlink, if it exists)
         // Please note that this will cause failures in Windows, which does not support symlinks.
         def symlinkLocation = destinationDir.toString() + "/hellbender-protected.jar"
+        def symlinkLocation2 = destinationDir.toString() + "/gatk-protected.jar"
+        exec {
+            commandLine 'ln', '-fs', archivePath, symlinkLocation
```

Comment:
Looks like I cannot force an overwrite.  That makes this a big pain.

No action.